guides/miner/hiveos.mdx

```diff
@@ -118,13 +118,13 @@ Now that all of our dependencies are installed, we can install quai-gpu-miner. T
     To download the binaries for the latest release into the `output` directory, run the following command:
 
     ```bash
-    sudo wget -P output https://github.com/dominant-strategies/quai-gpu-miner/releases/latest/download/quai-gpu-miner
+    wget -P output https://github.com/dominant-strategies/quai-gpu-miner/releases/latest/download/quai-gpu-miner
     ```
 
     This will create a built `quai-gpu-miner` file in the `output` directory. Make `quai-gpu-miner` executable by running:
 
     ```bash
-    chmod +x output/quai-gpu-miner
+    sudo chmod +x output/quai-gpu-miner
```

Comment:
I don't recommend sudo here. It's only relevant for the build script and its covered in that section

Reply:
Removed in [cea9168](https://github.com/dominant-strategies/quai-docs/pull/59/commits/cea91684e5c1365b50a9450a0696455bdcf0a0ef)